FilterViewController.m

```diff
@@ -125,28 +189,26 @@ - (void) createModalWithType:(CRUCellViewInteractionType)cellType section:(MHCol
     
 }
 
-- (void)saveChangesForCurrentSection{
+    }
```

Comment:
I'm not sure what happened here, the method looks like this:

- (void)saveChangesForCurrentSection{
    
    [self.tableView setBounces:YES];
    self.tableView.scrollEnabled = YES;
    
    if(self.currentModalType == CRUCellViewInteractionTextBox){
        
        [self resignFirstResponderWithClearOption:NO];
    }
    
    [self.currentSection saveChanges];
    
    //since checklist type uses real modal
    if(self.currentModalType != CRUCellViewInteractionCheckList){
    
        //currentSubViewControllerIndex is set as 0 by default in viewDidLoad
        UINavigationController *navigationController = self.childViewControllers[self.currentSubViewControllerIndex];
        [self removeHalfModalWithController:navigationController cgSize:self.view.bounds.size offSet:self.tableView.contentOffset];
     
    }
    else{
        self.presentedViewController.modalTransitionStyle = UIModalTransitionStyleCrossDissolve;
        [self dismissViewControllerAnimated:YES completion:nil];
    }
    
    self.modalCurrentlyShown = NO;
    //Reload the modal row to show results
    [self.tableView reloadRowsAtIndexPaths:@[self.currentRowPath] withRowAnimation:UITableViewRowAnimationNone];
    //Reload the header row of that modal so it's counter for # of selected items changes
    NSIndexPath *rootPath = [NSIndexPath indexPathForRow:self.currentSection.headerRowNum inSection:self.currentSection.returnManagerIndex];
    [self.tableView reloadRowsAtIndexPaths:@[rootPath] withRowAnimation:UITableViewRowAnimationNone];
    //manager header row num is always 0, reload this just in case it's a newly selected item for # change text
    rootPath = [NSIndexPath indexPathForRow:0 inSection:self.currentSection.returnManagerIndex];
    [self.tableView reloadRowsAtIndexPaths:@[rootPath] withRowAnimation:UITableViewRowAnimationNone];
    rootPath = nil;
}